WebJobs.Script.sln

```diff
@@ -250,6 +250,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Powershell", "H
 		sample\HttpTrigger-Powershell\run.ps1 = sample\HttpTrigger-Powershell\run.ps1
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "modules", "modules", "{23A9F11B-AD89-4164-BAD3-2B83B80A8173}"
+	ProjectSection(SolutionItems) = preProject
+		sample\HttpTrigger-Powershell\modules\Get-WorkerProcess.psm1 = sample\HttpTrigger-Powershell\modules\Get-WorkerProcess.psm1
```

Comment:
Please add a test to SamplesEndToEnd tests for your new sample. Take a look at the other tests - they'll enable you to verify the script log output etc.